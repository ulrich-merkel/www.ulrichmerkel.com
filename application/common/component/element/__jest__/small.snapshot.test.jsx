/* eslint-disable import/no-extraneous-dependencies, func-names */
import React from 'react';
import renderer from 'react-test-renderer';
import ElementSmall from '../small';

describe('common/component/element/small', function () {
    it('should render correctly', function () {
        const tree = renderer.create(
            <ElementSmall htmlElement='span' className='small'>
                Small Children
            </ElementSmall>
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});
