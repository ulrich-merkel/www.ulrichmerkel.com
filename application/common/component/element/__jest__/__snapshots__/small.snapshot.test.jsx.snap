// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common/component/element/small should render correctly 1`] = `
<span
  className="c-type--small small"
>
  Small Children
</span>
`;
